### DISCUSSION TITLE
Missing environment variables

### DISCUSSION THREAD

**Discussion by groovy-grapes** [sleeyax/aternos-discord-bot#88] [Q&A]
So in trying to run the bot on termux and when i use the DISCORD_TOKEN="" MONGO-BD-URI="" ./aternos-discord-bot command thingy (i did add the discord token) i just get the missing environment variables thingy

**Comment by sleeyax** [comment]
You typed mongo-db wrong. Make sure at least the discord token is actually set correctly.
